doc/user/content/sql/show-materialized-views.md

```diff
@@ -11,12 +11,14 @@ in Materialize.
 
 ## Syntax
 
-{{< diagram "show-materialized-views.svg" >}}
+```mzsql
+SHOW MATERIALIZED VIEWS [ FROM <schema_name> ] [ IN <cluster_name> ]
+```
 
-Field | Use
-------|-----
-_schema&lowbar;name_ | The schema to show materialized views from. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
-_cluster&lowbar;name_ | The cluster to show materialized views from. If omitted, materialized views from all clusters are shown.
+Option                      | Description
+----------------------------|------------
+**FROM** <schema_name>      | If specified, only show materialized views from the specified schema. Defaults to first resolvable schema in the search path. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
```

Comment:
It is a little repetitious with the 2 "specified" ... but eh.
I tried other variations like "If included, ... the specified schema" or "If specified, ... the given schema." or "If specified ... <schema_name>" but I think specified works better than those.  Maybe during other batches/sweeps, something better will pop into mind.